src/mlfe_parser.yrl

```diff
@@ -96,14 +96,14 @@ type_import -> use module_fun:
 
 module_def -> module atom : {module, '$1'}.
 
-type_vars -> type_var : ['$1'].
-type_vars -> type_var type_vars : ['$1'|'$2'].
+type_vars -> type_expr : ['$1'].
```

Comment:
Should we maybe change from `type_var` to something else like `type_expressions` or something along those lines?  Minor point but might alleviate confusion later :)